index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/7415ea4535.js" crossorigin="anonymous"></script>
+<body>
+
+    <header>
```

Comment:
- [ ] As mentioned by the previous reviewer, you need to lay out the elements on the page using Flexbox because it gives a lot of options to align your elements. To learn more you can check this [page](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) :+1: